scripts/diagnose-and-requeue.sh

```
@@ -164,7 +174,7 @@ else
         aws sqs send-message \
           --queue-url "$SQS_QUEUE_URL" \
           --message-body "{\"job_id\":$NEW_JOB_ID,\"note_id\":\"$note_id\",\"filename\":\"$ESCAPED_FILENAME\",\"import_id\":$id,\"attempt_number\":1,\"retry\":true}" \
-          --region eu-north-1 2>&1 > /dev/null
+          --region "$AWS_REGION" 2>&1 > /dev/null
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)